src/mixing.html

```diff
@@ -99,9 +124,9 @@ <h3 id="mixing_msup_notation">Same notation, different meanings</h3>
 &lt;/msup>
 </pre>
 
+ 
  </section>
  </section>
-
  
  <section>
   <h3 id="mixing_semantic_annotations">Annotation Elements</h3>
```

Comment:
yes, alhough I want it to be the `<semantics` annoaion rather than `intent=` I'll make it `mixing_annotation_elments` (actually we should probably change the `mixing_` prefix as well but that would chang refrences over he document (and the notes document) so leaving that for now to keep the diff here localised.